src/olympia/devhub/forms.py

```diff
@@ -730,14 +730,19 @@ class AdminForm(happyforms.ModelForm):
     _choices = [(k, v) for k, v in amo.ADDON_TYPE.items()
                 if k != amo.ADDON_ANY]
     type = forms.ChoiceField(choices=_choices)
+    reputation = forms.ChoiceField(
+        label=_(u'Reputation'),
+        choices=[('', '')] + [(i, unicode(i)) for i in range(-3, 4)])
```

Comment:
The choices are fixed between -3 and +4?  Are they arbitrary values or is there logic behind it?